src/services/StudentService.js

```diff
@@ -381,6 +384,27 @@ const stageColumn = {
   },
 };
 
+const EvaluationColumn = {
+  name: "evaluation",
+  label: "Evaluation",
+  options: {
+    filter: false,
+    sort: true,
+    display: permissions.updateStudentName.indexOf(user) > -1 ? true : false,
```

Comment:
`? true : false` is not needed here; if `permissions.updateStudentName.indexOf(user) > -1` evaluates to `true`, then `display` will be set to `true`; if it's `false`, then `display` will be set to `false` so since these are the only possibilities you can get from the inequality, `display` will always be set to whatever Boolean value you get from `permissions.updateStudentName.indexOf(user) > -1`.  But maybe just use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here?
```suggestion
    display: permissions.updateStudentName.includes(user)
```